go/vt/sqlparser/ast.go

```diff
@@ -2223,6 +2223,12 @@ type (
 		alternative  Expr // this is what will be used to Format this struct
 	}
 
+	// JSONUtilityExpr represents set of JSON utility functions
+	// More information on https://dev.mysql.com/doc/refman/8.0/en/json-utility-functions.html
+	JSONUtilityExpr struct {
+		Name      ColIdent
+		StringArg Expr
+	}
```

Comment:
I think @GuptaManan100 is right. Even though the three different utility functions (`JSON_PRETTY`, `JSON_STORAGE_SIZE`, `JSON_STORAGE_FREE`) have the same signature, it would be more readable/debuggable to have them as separate AST nodes or have an identifiable name (such as an enum).